### DISCUSSION TITLE
Creazione autofattura TD01

### DISCUSSION THREAD

**Discussion by FabioArmonia** [fattureincloud/api#370] [Q&A] [answered]
Buonasera,
Sto utilizzando il vostro SDK di typescript e ho bisogno di creare un'autofattura per un fornitore.

Ho impostato come entity il fornitore ma la fattura viene generata con lui come destinatario mentre io avrei bisogno che l'azienda dell'account connesso sia il destinatario (e quindi il cessionario/committente) e il fornitore come tale.

Come posso raggiungere questo obiettivo? Quali campi devo compilare?

**Comment by emajo** [comment]
Ciao @FabioArmonia, il mio consiglio per ottenere un esempio di richiesta cucito su misura per il suo caso è quello di creare un documento dalla nostra interfaccia web e successivamente fare un chiamata API GET per quel documento, per vedere esattamente come sono stati popolati i campi per il suo caso.

Grazie,
Emanuele.

**Reply by FabioArmonia** [reply]
Buongiorno Emanuele,

Ho provato a creare un'autofattura dalla dashboard e il risultato è lo stesso di quando la creo tramite API.
Dalle specifiche e gli esempi che mi hanno fornito il cessionario/committente dovrebbe essere l'azienda che crea l'autofattura e dovrebbe essere quindi posizionata sotto la voce "Destinatario" (dato che la nota autogenerata suggerisce proprio questo), mentre creandola da API o piattaforma il "Destinatario" risulta essere il fornitore. 

È possibile sia un bug o un errore delle specifiche che mi sono arrivate? A chi posso rivolgermi nel caso di un errore nella generazione dell'autofattura?

**Reply by valmoz** [reply]
Buongiorno @FabioArmonia 
in tal caso il suggerimento è quello di rivolgersi al supporto clienti sulla pagina principale di Fatture in Cloud, che saprà supportarla al meglio nella creazione della fattura tramite pagina web. Le suggerisco di indicare che ha parlato prima con noi tramite Community, e magari di inserire il collegamento a questa discussione.

Una volta risolto il problema della generazione dell'autofattura, potrà procedere ad adottare il procedimento suggerito dal collega.

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @FabioArmonia è riuscito a risolvere?

**Reply by FabioArmonia** [reply, marked as answer]
Buongiorno,
il problema è stato risolto. 
Uno sviluppatore del supporto ha modificato il template che stavamo utilizzando invertendo i dati che venivano mostrati.

Grazie della collaborazione
